### PR TITLE
graph-services to be auto-released on merge to master

### PR DESCRIPTION
**Pull request by jachro**
Graph Services have a comprehensive set of unit and acceptance tests as well as every merge to master is reviewed. Therefore, the release process should be automatic and no manual steps should be required. This PR introduces changes required to make the release like that.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 dist: xenial
 branches:
   only:
-    - master
-    - /^\d+\.\d+\.\d+(-\S*)?$/
+    - /^\d+\.\d+\.\d+$/
```
Comment (rokroskar) [inline .travis.yml]: only on tags?

Comment (jachro) [reply .travis.yml]: yes, as effectively every successful build on master is a release you can pick up and drop to prod :)

Comment (jachro) [reply .travis.yml]: or actually, the master still need to be there as the release process won't get kicked off :)

Comment (rokroskar) [reply .travis.yml]: yes, that's my point :)

Comment (rokroskar) [reply .travis.yml]: but I've been wrong about the magic of travis before as you well know... so I wasn't sure if you had some new trick :)

Comment (jachro) [reply .travis.yml]: thankfully no tricks :)

**Pull request merged**